### PR TITLE
Germline naming

### PR DESCRIPTION
**Pull request by shilpigoeldev**
Have corrected rubocop issues which were straightforward and not needed logic/code rewriting (not in scope of this work) .

### CODE REVIEW

**Review by FrancescoNea — APPROVED** [review]
All tests pass. I have added disablers for some Rubocop complaints.
These complaints are mostly about classes, couple of methods and some lines being too long.
As I think that, for the moment being, these complaints are low priority, I am happy to approve this PR.

File: lib/import/utility/extraction_utilities.rb [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+        GENE_LOC = "#{GENE_ONLY} ?(?<protein>\(p\.[^\)\(]*\))?".freeze
+        BRCA_BASE = 'br?c?a?'.freeze
+        BRCA     = BRCA_BASE + '(?<brca>1|2)'.freeze
+        EXON_LOC = 'exons? (?<exons>\d+[a-z0-9]*(?:-\d+[a-z0-9]*)?)'.freeze
+        VARIANT  = '(?:familial)?(?<variantclass>(?: likely) pathogenic)?' \
+        ' BRCA(?<brca>1|2) (?:mutation|sequence variant)'.freeze
+        ZYGOSITY = '(?<zygosity>[a-z]+)zygous'.freeze
+        GENE_LOCATION = '(?<location>c\.[^ \.]+) ?(?<protein>\(p\.[^)]*\))?'.freeze
+        EXON_LOCATION = 'exons? (?<exons>\d+[a-z0-9]*(?:-\d+[a-z0-9]*)?)'.freeze
+        LOCATION_REGEX = /(#{GENE_LOCATION}(?: in (?:brca(?:1|2) )?#{EXON_LOCATION})?|#{EXON_LOCATION})/i
```
Comment (ollietulloch) [inline lib/import/utility/extraction_utilities.rb:20]: @shilpigoeldev could you look at using the `/x` modifier to split the regexp over multiple lines? [Example here](https://stackoverflow.com/questions/3762183/split-ruby-regex-over-multiple-lines).

There are additional lines in this file that should also be corrected

File: lib/import/germline/genotype.rb [hunk 2]
```diff
@@ -0,0 +1,505 @@
...
+      end
+
+      def get(key)
+        @attribute_map[key]
+      end
+
+      attr_reader :attribute_map, :pseudo_id1, :pseudo_id2, :raw_record
+
+      # ------------------------- Comparators ------------------------------
+      def similar_record(other_genotype, field_list, strict = true)
```
Comment (ollietulloch) [inline lib/import/germline/genotype.rb:118]: Please ue kwargs when defining method with boolean argument - [example here](https://stackoverflow.com/questions/67391214/style-optionalbooleanparameter-use-keyword-arguments-when-defining-method-with)

Remember to change all calls of this method

File: lib/import/utility/extraction_utilities.rb [hunk 3]
```diff
@@ -0,0 +1,152 @@
...
+      #       packages do. This applies mostly to standarising c. and p. forms and catching
+      #       some edge cases/unusual mutations which would currently be rejected.
+
+      # Provide utilities to extract c., p., and exon-based location information from
+      # raw strings
+      class LocationExtractor < CommonUtility
+        CDNA = '\\*?\\d+((_|-|\\+)\\d+)? ?([a-z]+(?:>|<)[a-z]+|(del|dup|ins)' \
+        ' ?(?:[atcg]*|[0-9]*))*(?=(?:\\s|$|\\()|,)'.freeze
+
+        def initialize(write_csv = false)
```
Comment (ollietulloch) [inline lib/import/utility/extraction_utilities.rb:66]: Please ue kwargs when defining method with boolean argument - [example here](https://stackoverflow.com/questions/67391214/style-optionalbooleanparameter-use-keyword-arguments-when-defining-method-with)

Remember to change all calls of this method

**Review by ollietulloch — CHANGES_REQUESTED** [review]
Hi @shilpigoeldev ,

A couple of things to adjust for rubocop, please see comments. I realise that these are inherited issues from existing code, but it'd be good to take the chance to fix them.

Thanks,
Ollie

**Comment by shilpigoeldev** [comment]
@ollietulloch I have now worked on the review comments and rematched the counts , thank you 🙏

**Pull request merged**